src/picknik_ur_site_config/objectives/get_hinge_from_user.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<root BTCPP_format="4" main_tree_to_execute="GetHingeFromUser">
+  <!-- ////////// -->
+  <BehaviorTree ID="GetHingeFromUser" _description="" _favorite="false" _hardcoded="false">
```

Comment:
for consistency, call it "Get Hinge From User" with spaces.

Reply:
deleted this file as it was no longer used.